internal/controllers/clusterextension_controller_test.go

```diff
@@ -39,61 +38,7 @@ func TestClusterExtensionDoesNotExist(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestClusterExtensionResolutionFails(t *testing.T) {
```

Comment:
We still need this test. We just expect the failure to show up as `Progressing=True, Reason=Retrying` with a message that says there was a resolution failure.

Reply:
Fixed in https://github.com/operator-framework/operator-controller/commit/c214b0b405df178c7bb1ce9a5d9da31cec7ac332